src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -1,62 +1,156 @@
-import { Spacer, StackableBorderBox } from "@artsy/palette"
-import { artistResponse } from "Apps/__test__/Fixtures/MarketInsights"
-import React from "react"
+import React, { SFC } from "react"
+
+import { Box, Sans, Spacer, Spinner, StackableBorderBox } from "@artsy/palette"
+import { ArtistInfo_artist } from "__generated__/ArtistInfo_artist.graphql"
+import { ArtistInfoQuery } from "__generated__/ArtistInfoQuery.graphql"
+import { ContextConsumer } from "Artsy"
+import { Mediator } from "Artsy/SystemContext"
+import FollowArtistButton from "Components/FollowButton/FollowArtistButton"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { EntityHeader } from "Styleguide/Components/EntityHeader"
+import { get } from "Utils/get"
 
 import {
-  ArtistBio,
-  MarketInsights,
-  MarketInsightsProps,
-  SelectedExhibitions,
-  SelectedExhibitionsProps,
+  ArtistBioFragmentContainer as ArtistBio,
+  MarketInsightsFragmentContainer as MarketInsights,
+  SelectedExhibitionFragmentContainer as SelectedExhibitions,
 } from "Styleguide/Components"
-import { EntityHeader } from "Styleguide/Components/EntityHeader"
 
-interface ArtistInfoProps
-  extends MarketInsightsProps,
-    SelectedExhibitionsProps {
-  name: string
-  bio: string
-  credit?: string
+interface ArtistInfoProps {
+  artist: ArtistInfo_artist
+  user: User
+  mediator?: Mediator
 }
 
-export class ArtistInfo extends React.Component<ArtistInfoProps> {
-  render() {
-    // FIXME: Fetch from relay
-    const props = {
-      href: "/artist/francesca-dimattio",
-      imageUrl: "https://picsum.photos/110/110/?random",
-      name: "Francesca DiMattio",
-      meta: "American, b. 1979",
+export const ArtistInfo: SFC<ArtistInfoProps> = props => {
+  const imageUrl = get(props, p => p.artist.image.cropped.url)
+
+  return (
+    <>
+      <StackableBorderBox p={2} flexDirection="column">
+        <EntityHeader
+          name={props.artist.name}
+          meta={props.artist.formatted_nationality_and_birthday}
+          imageUrl={imageUrl}
+          FollowButton={
```

Comment:
@oxaudo - This now accepts a component, which can be whatever

Reply:
Cool. Now I just need to find/create one for follow partner.